File: Programas em C/Ex11.c
#include<stdio.h>
int main(int argc, char const *argv[])
{
    int x, a;
    printf("Digite um valor: ");
    scanf("%i", &a);

    printf("Digite outro valor: ");
    scanf("%i", &x);

    if(x < a){
        printf("O maior numero e: %i", a);
    }
    else if(x > a){
        printf("O maior numero e: %i", x);
    }
    else{
        printf("Os numeros sao identicos!");
    }
    return 0;
}

File: Programas em Phyton/Ex11.py
v1 = float(input('Digite o primeiro valor: '))
v2 = float(input('Digite o segundo valor: '))
if(v1>v2):
    print(f"{v1:.0f} é maior")
elif(v2>v1):
    print(f"{v2:.0f} é maior")
else:
    print("São idênticos!")